build/packages/rpm/client/pmm2-client.spec

```
@@ -14,7 +14,9 @@ Summary:        PMM-agent
 
 BuildRequires:    systemd
 BuildRequires:    pkgconfig(systemd)
-Requires:         perl
+%if 0%{?rhel} && 0%{?rhel} == 9
```

In general I would suggest this
```suggestion
%if 0%{?rhel} && 0%{?rhel} >= 9
```